docs/conf/prague/2020/welcome-wagon.rst

```diff
@@ -74,6 +73,13 @@ To attend the moderated Q&A after each talk, go to **Sessions** and select the *
 - Make sure the **Session** toggle is on for the Chat, so your question will be seen by the moderator.
 - Due to time constraints, we might not be able to get to all of your questions. If your question isn’t asked live, chat with the speaker later and make a connection!
```

Comment:
I know it isn't the intent of your PR, but add `Q:` to add questions? :-p

Reply:
Using "Q:" is the first bullet in this section—just inside the GitHub diff folding, of course

Reply:
:facepalm: sorry @ddbeck my bad